src/formatDate.js

```diff
@@ -8,7 +8,31 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const splittedDate = date.split(fromFormat[fromFormat.length - 1]);
```

Comment:
Consider using a more descriptive variable name instead of `splittedDate`. Variable names should be clear and self-explanatory, something like `dateParts` might convey the purpose more clearly.